### DISCUSSION TITLE
want to change nav.group

### DISCUSSION THREAD

**Discussion by shruko** [Z3d0X/filament-logger#48] [Q&A]
is there any method to change the Navigation Group

**Comment by fsamapoor** [comment]
The package uses `__('filament-logger::filament-logger.nav.group')` as the navigation group.
So right now, basically you have two options:

1. Publish the package's translation files and overwrite the `nav.group` translation to your needs.
2. Extend the package's `ActivityResource` class, overwrite the `getNavigationGroup` method, and replace the `activity_resource` config entry with the namespace of the extended class.

**Reply by esemangrafico** [reply]
I take the first option, just publish the Translations and change the group! It worked perfect! Thanks! 👍🏼
